tests/src/Interop/DllImportAttribute/DllImportPath/DllImportPathNative.cpp

```diff
@@ -2,20 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <locale.h>
-#include <xplatform.h>
+#include "xplatform.h"
 
-#pragma warning( push )
-#pragma warning( disable : 4996)
-
-extern "C" DLL_EXPORT int __cdecl Sum(int a, int b)
+extern "C" DLL_EXPORT int STDMETHODCALLTYPE GetZero()
```

Comment:
Since 0 tends to be a common indication of success, I would suggest a more non-standard number. Say 37 or something that doesn't have a potentially ambiguous meaning. Entirely up to you to decide, but -1, 0, and 1 all are common error values.

Reply:
The value here doesn't actually matter at all since we're just testing the DLL resolution. We don't even check the value on the managed side.

Reply:
Ah okay.